backend/src/packages/journal-entries/journal-entry.service.ts

```diff
@@ -46,8 +50,20 @@ class JournalEntryService implements Service {
     return Promise.resolve(null);
   }
 
-  public delete(): ReturnType<Service['delete']> {
-    return Promise.resolve(true);
+  public async delete(
+    id: number,
+    user: UserAuthResponseDto,
+  ): ReturnType<Service['delete']> {
+    // const journal = await this.find(); //TODO find(id)
+    const journal = { id: 1, userId: 32 };
```

Comment:
Remove mock data

Reply:
@v0ldemar01 but for now we don`t have userId in journal model

Reply:
But we will have it if you are linked to it

Reply:
@v0ldemar01 it`s like if you are loged in and have token, and you know link to backend part without it you can delete any user`s journal entry